SETUP.md

```diff
@@ -0,0 +1,101 @@
+# Setup Instructions
+
+To participate in the workshop, you will need access to the software described below. In addition, you will need an up-to-date web browser.
+
+## The Bash Shell
+
+Bash is a commonly-used shell that gives you the power to do simple
+tasks more quickly.
+
+### Windows
+
+<a href="https://www.youtube.com/watch?v=339AEqk9c-8">Video Tutorial</a>
+
+1. Download the Git for Windows <a href="https://git-for-windows.github.io/">installer</a>.
```

Comment:
Why are we installing git? I don't think we need git for the workshop, but we do need PuTTY for SSH sessions.

Reply:
Git for Windows includes Git Bash-- "Git BASH Git for Windows provides a BASH emulation used to run Git from the command line."

Reply:
I think Ben is right. We are note doing a "programming" course. The shell component really has to be get your feet wet.  

For windows just stick with PuTTY
https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html
or simply Google putty, first result.Its nice and small and we can even have it on a usb stick if anyone doesn't download it and needs it.